python/cross_service/aurora_item_tracker/templates/work_items.csv

```diff
@@ -0,0 +1,4 @@
+Description,Guide,Status,User name,Archived
+{%- for work_item in work_items %}
+{{work_item["description"]}},{{work_item["guide"]}},{{work_item["status"]}},{{work_item["username"]}},{{'Yes' if work_item["archived"] else 'No'}}
```

Comment:
This probably doesn't quote or escape correctly - could it switch to using csv.writer from PEP305?

Reply:
Oh, all right...*fine*. 🙄😏